javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,20 @@
+let fullName = 'Logan Taehwan Lee';
+console.log('value of fullName:', fullName);
+
+let isCool = true;
+console.log('value of isCool:', isCool);
+
+let totalPets = 0;
+console.log('value of totalPets:', totalPets);
```

Comment:
These should be using `const` instead of `let`. Since we don't plan on reassigning these variables to any other value, they should use the `const` keyword. `let` is only used for variables that we know their value will change